Description: Help 4Pane find its F1 Help files
Author: David Hart (david@4Pane.co.uk)
diff --git a/Configure.cpp b/Configure.cpp
index bf612c9..0db2e2a 100644
--- a/Configure.cpp
+++ b/Configure.cpp
@@ -392,6 +392,7 @@ FileData stat3((HELPDIR = wxGetApp().StdDocsDir) + wxT("/About.htm")); if (stat3
 FileData stat4((HELPDIR = wxT("/usr/local/share/4Pane/doc/")) + wxT("About.htm")); if (stat4.IsValid()) return true;// Or hard-wire this (might be needed in a livecd situation)
 FileData stat5((HELPDIR = wxT("/usr/share/doc/4pane/")) + wxT("About.htm")); if (stat5.IsValid()) return true;      // Ditto in debianish systems
 FileData stat6((HELPDIR = wxT("/usr/share/doc/4Pane/")) + wxT("About.htm")); if (stat6.IsValid()) return true;      //  or almost...
+FileData statofficial((HELPDIR = wxT("/usr/share/doc/4Pane/html/")) + wxT("About.htm")); if (statofficial.IsValid()) return true; //  or in official builds
 FileData stat7((HELPDIR = wxT("/usr/doc/4Pane/")) + wxT("About.htm")); if (stat7.IsValid()) return true;    // Ditto in slackware (Sigh! And even that's a symlink)
 FileData stat8((HELPDIR = GetCwd()+wxT("/doc/")) + wxT("About.htm")); if (stat8.IsValid()) return true;     // If still not, look in ./docs
 FileData stat9((HELPDIR = HOME+wxT("/doc/")) + wxT("About.htm")); if (stat9.IsValid()) return true;         // 1 last chance. Try in HOME
